src/Extension.ml

```diff
@@ -116,6 +116,8 @@ let activate (extension : Vscode.ExtensionContext.t) =
   Vscode.ExtensionContext.subscribe extension
     (Vscode.Commands.register ~command:selectSandboxCommandId
        ~handler:(selectSandbox instance));
+  Vscode.ExtensionContext.subscribe extension
+    (Vscode.Disposable.create ~dispose:(fun () -> Instance.stop instance));
```

Comment:
Could you add an intermediate function to `Instance`:

```ocaml
val disposable : Instance.t -> Disposable.t
```

If we follow this convention, it should be easier for us to remember to dispose of things a bit more diligently.

Reply:
Makes sense. Just updated.